src/Utilities/ErrorHandling/AbortWithErrorMessage.cpp

```diff
@@ -16,6 +17,19 @@
 namespace {
 struct FillBacktrace {};
 
+// Convert the address to the virtual memory address inside the
+// library/executable. This is the address that addr2line and llvm-addr2line
+// expect.
+size_t convert_to_virtual_memory_address(const size_t addr) {
```

Comment:
can you pass the address as a `void *`? I understand the reluctance, but I find passing memory addresses as `size_t` and reinterpret-casting more confusing than the C-style idioms.